### PR TITLE
PR: Add jupyterlab-translate package

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/jupyterlab-translate/build.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$PYTHON -m pip install . -vv
+npm install gettext-extract
```
Comment (scopatz) [inline recipes/jupyterlab-translate/build.sh:2]: If this is jlab ext, please see https://conda-forge.org/docs/maintainer/knowledge_base.html#jupyterlab-extensions

Comment (goanpeca) [reply recipes/jupyterlab-translate/build.sh:2]: Nope this is not a JLab Extension, but it needs a package available on NPM

Comment (goanpeca) [reply recipes/jupyterlab-translate/build.sh:2]: But I could follow that docs, if it still applies?

Comment (scopatz) [reply recipes/jupyterlab-translate/build.sh:2]: So those only work if the npm package will get picked up by Jlab on the user's machine.  I am not sure how this project works, but I think the npm install here will only install place it into conda-build's working directory, not the PREFIX. So the gettext-extract won't make it into the package

Comment (goanpeca) [reply recipes/jupyterlab-translate/build.sh:2]: Right. What is the right way to bundle an npm package?

Comment (scopatz) [reply recipes/jupyterlab-translate/build.sh:2]: I think you do a global install

Comment (goanpeca) [reply recipes/jupyterlab-translate/build.sh]: Done 🤷

**Comment by goanpeca** [comment]
@scopatz ok, I found a better way of adding the package, so will do in a sec

**Comment by scopatz** [comment]
Should this be closed then?

**Comment by goanpeca** [comment]
Nope I will push an update.

**Comment by goanpeca** [comment]
This one is ready @scopatz :-)

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**